toolkit/analysis_and_visualization/classification/assets/scripts/pagination.js

```diff
@@ -0,0 +1,1125 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-empty */
+/*global jQuery, define*/
+
+/*
+ * pagination.js 2.1.5
+ * A jQuery plugin to provide simple yet fully customisable pagination.
+ * https://github.com/superRaytin/paginationjs
+ *
+ * Homepage: http://pagination.js.org
+ *
+ * Copyright 2014-2100, superRaytin
+ * Released under the MIT license.
+ */
+
+(function (global, $) {
```

Comment:
Codacy found an issue: [Function has a complexity of 5. Maximum allowed is 4.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)